src/main/java/org/opensearch/ad/model/ADTaskAction.java

```diff
@@ -28,5 +28,9 @@
 
 public enum ADTaskAction {
     START,
-    STOP
+    FINISHED,
+    CANCEL,
+    NEXT_ENTITY,
+    PUSH_BACK_ENTITY,
+    CLEAN_RUNNING_ENTITY
```

Comment:
It would be great if we can have some comments explaining those states.

Reply:
Sure, I add comments in `ForwardADTaskTransportAction`, will copy comments here.